nixos/modules/system/boot/loader/gummiboot/gummiboot.nix

```diff
@@ -31,7 +31,9 @@ in {
     };
 
     timeout = mkOption {
-      default = null;
+      default = if (config.boot.loader.timeout != null) then
```

Comment:
Was going to test this PR, however here does it mean a timeout of either null or 0 means no entry in gummiboot? Note that unsetting the timeout in gummiboot means the entry is executed immediately, afaik, which is different than the description of `boot.loader.timeout`.